.github/workflows/release.yml

```diff
@@ -519,6 +520,12 @@ jobs:
           timestampUrl: 'http://timestamp.digicert.com'
           folder: appdir/Cryptomator
           recursive: true
+      - name: Generate license
+        run: >
+          mvn -B license:add-third-party
+          "-Dlicense.thirdPartyFilename=license.rtf"
+          "-Dlicense.fileTemplate=dist/win//resources/licenseTemplate.ftl"
+          "-Dlicense.outputDirectory=dist/win/resources"
```

Comment:
the license is not mentioned in `actions/upload-artifact`, so I don't really understand, how it finds its way to the exe bundle...?

Reply:
see below (https://github.com/cryptomator/cryptomator/pull/2072#discussion_r813660066)